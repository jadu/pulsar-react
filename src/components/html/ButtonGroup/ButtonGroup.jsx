import React from 'react';

export default class ButtonGroup extends React.Component {

  static defaultProps = {
    className: 'btn__group'
  };

  render() {
    let {
      children,
      ...props
    } = this.props;

    return (
      <>
        <div
          {...props}
        >
          {children}
        </div>
      </>
    );
  }
}
