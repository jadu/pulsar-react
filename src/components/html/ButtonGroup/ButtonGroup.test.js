'use strict';

import React from 'react';
import { render } from 'enzyme';

import ButtonGroup from './ButtonGroup';

it('renders the button group', () => {
  const tree = render(
    <ButtonGroup>foo</ButtonGroup>
  );
  expect(tree).toMatchSnapshot();
});
