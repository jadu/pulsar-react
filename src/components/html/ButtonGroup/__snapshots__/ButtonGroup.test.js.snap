// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the button group 1`] = `
<div
  class="btn__group"
>
  foo
</div>
`;
